examples/modify-pin/README.md

```diff
@@ -0,0 +1,60 @@
+# Setting and modifying PIN codes
+
+This example shows you the functionality and usage of the following four commands:
+* [set_pin](#set_pin-command) - Sets a new PIN code
+* [change_pin](#change_pin-command) - Changes current PIN code to a new one
+* [unlock_pin](#unlock_pin-command) - Removes PIN code using the corresponding PUK
```

Comment:
"Removing PIN code" might be misleading. I would phrase it like "Deactivating the PIN authentication..."